FactoryMethod/src/main/java/com/ben/abstractnotification/AndroidPhone.java

```diff
@@ -0,0 +1,16 @@
+package com.ben.abstractnotification;
+
+public class AndroidPhone extends Phone{
+
+    @Override
+    public void getRam() {
```

Comment:
I think your `getRam` method here should return `String` instead of printing it out

Reply:
Yeah you're right.